### PR TITLE
Backend/facilitate postal codes call. Remove throttle

### PR DESCRIPTION
**Pull request by garrickhenne**

### CODE REVIEW

**Comment by garrickhenne** [comment]
Not going to merge this branch into main for the time being, feels too risky. We can decide to present from this branch if everything checks out correctly.

File: frontend/src/routes/Map.jsx [hunk 1]
```diff
@@ -66,121 +67,87 @@ const Map = () => {
...
   };
 
   return (
     <div>
-
       <ReactMapGL
         {...viewport}
         mapStyle="mapbox://styles/mapbox/streets-v12"
-        onMove={evt => setViewport(evt.viewState)}
+        onMove={evt => setViewport((prevState) => evt.viewState)}
```
Comment (robertshum) [inline frontend/src/routes/Map.jsx:129]: do we need prevState here?

**Review by robertshum — COMMENTED** [review]
just a minor question in map.jsx but it looks good, sir.

File: frontend/src/routes/Map.jsx (hunk 1 above)
Comment (garrickhenne) [reply frontend/src/routes/Map.jsx]: good find!

**Pull request merged**